test/e2e/util/util.go

```diff
@@ -2073,3 +2087,71 @@ func EnsureCustomTolerations(t *testing.T, ctx context.Context, client crclient.
 		}
 	})
 }
+
+// EnsureCreateClusterWithDisabledCapabilities implements a test that creates a cluster
```

Comment:
The doc for this function doesn't seem accurate neither the name, 

// EnsureImageRegistryCapabilityDisabled validates the expectations for when ImageRegistryCapability is Disabled
EnsureImageRegistryCapabilityDisabled